cloudmarker/test/test_util.py

```diff
@@ -71,6 +80,28 @@ def test_parse_cli_args_config(self):
         args = util.parse_cli(['--config', 'baz.yaml', 'qux.yaml'])
         self.assertEqual(args.config, ['baz.yaml', 'qux.yaml'])
```

Comment:
I think there should also be a test for default config with just `-c` option like

```
    def test_parse_cli_args_no_config(self):
        # Short option.
        args = util.parse_cli(['-c'])
        self.assertEqual(args.config, [])
```

Reply:
Yes, I like this suggestion. I have added a new test method here: [test_parse_cli_args_no_config](https://github.com/cloudmarker/cloudmarker/pull/90/files#diff-45528363d1b141abd9b51e1eb4f6e297R75).